angrmanagement/ui/workspace.py

```diff
@@ -143,6 +144,39 @@ def on_function_tagged(self):
     # Public methods
     #
 
+    def new_disassembly_view(self):
+        """
+        Add a new disassembly view into the
+        central_widget and rename the views
+        accordingly
+
+        :return:    None
+        """
+
+        # Replace first disassembly view to appropriate naming convention 
+        if self.view_manager.disas_view_counter == 1:
+            for view in self.view_manager.docks:
+                if view.windowTitle() == 'Disassembly':
```

Comment:
Let's not rename existing tab titles. How about we always name the first Diasssembly View as "Disassembly-A?"